pkg/ibm_plugin/sdk/vpc.go

```diff
@@ -26,12 +26,18 @@ import (
 
 const vpcType = "vpc"
 
+const SharedVPC = "shared"
```

Comment:
Does this need to be exported?

Reply:
Yes, it us used in plugin.go

Reply:
But they're both in the ibm package so i don't think it needs to be exported right?

Reply:
It still needs to be exported since it's in the context of ibm_plugin/sdk . 
`sdk.sharedVPC` doesn't seem to work, neither does sharedVPC..

Reply:
Got it no worries then

Reply:
I raised an issue related to this #318. Let's not fix it in this PR but just wanted to let you know.